streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableImpl.java

```diff
@@ -155,6 +160,37 @@ public void to(Serde<K> keySerde, Serde<V> valSerde, String topic) {
     }
 
     @Override
+    public KTable<K, V> print() {
+        return print(null, null);
+    }
+
+    @Override
+    public KTable<K, V> print(Serde<?> keySerde, Serde<?> valSerde) {
+        String name = topology.newName(PRINTING_NAME);
+        topology.addProcessor(name, new KStreamPrinter<>(keySerde, valSerde), this.name);
```

Comment:
If we do not return the object (see the general comment), then we can just call corresponding KStreamImpl.xx functions.